metricbeat/module/docker/network/_meta/fields.yml

```diff
@@ -4,7 +4,7 @@
     Network metrics.
   release: ga
   fields:
-
+    # network.interface?
```

Comment:
I would be in favour of leaving here. We don't know how it would be added to ECS: Will it be a list of keywords (e.g. eth0) or a list of objects, where the `name:eth0` is just one of the attributes?

Reply:
I see, it could be more then just the name? Then let's skip it for now.

Reply:
Metadata like bytes sent/received, associated IP, color ;-)